python/src/setup.py

```diff
@@ -27,7 +27,7 @@
 # To debug, set DISTUTILS_DEBUG env var to anything.
 setuptools.setup(
     name="GoogleAppEngineMapReduce",
```

Comment:
Do we need to change this name to something other than the main repo name to avoid conflicts of some sort?